app/Http/Controllers/ProjectController.php

```diff
@@ -51,16 +51,23 @@ public function index(Request $request)
      */
     public function store(Request $request)
     {
-        $project = new Project([
-          'name' => $request->get('name'),
-          'information' => $request->get('information'),
-          'deadline' => $request->get('deadline'),
-          'type' => $request->get('type'),
-          'status' => $request->get('status')
+      $rules = [
+      'name' => 'required|unique:member|regex:/^[a-zA-Z0-9-. ]+$/u|max:10',
```

Comment:
This is the project controller, yet you are declaring a validation rule that checks for a `unique:member`